README.md

```diff
@@ -31,35 +31,51 @@ optional arguments:
-     * `POKEBALL_CONTINUE_THRESHOLD`: `Integer`, when the observed pokeball count reaches this amount, stop farming and go back to catching pokemon.
-     * `FARM_IGNORE_POKEBALL_COUNT`: `Boolean`, Whether to include this ball in counting. Same goes for `GREATBALL`, `ULTRABALL`, and `MASTERBALL`. Masterball is ignored by default.
-     * `FARM_OVERRIDE_STEP_SIZE`: `Integer`, When it goes into farming mode, the bot assumes this step size to potentially speed up resource gathering. _This might lead to softbans._ Setting to `-1` disables this feature. Disabled by default for safety.
-     * If `EXPERIMENTAL` OR `CATCH_POKEMON` are false, this configuration will disable itself.
+   * `SPIN_ALL_FORTS` [Experimental] will try to route using google maps(must have key) to all visible forts, if `SKIP_VISITED_FORT_DURATION` is set high enough, you may roam around forever.
+   * `CATCH_POKEMON` Allows you to disabling catching pokemon if you just want to mine for the forts for pokeballs
+* `EGG_INCUBATION`
+   * `ENABLE` enables automatic use of incubators (default: true)
+   * `USE_DISPOSABLE_INCUBATORS` enables use of disposable (3-times use) incubators (default: false)
+   * `BIG_EGGS_FIRST` incubate big eggs (most km) first (default: true)
+* `POKEMON_CLEANUP`
+   * `KEEP_POKEMON_NAMES` Names of pokemon you want the bot to hold regardless of IV/CP
+   * `THROW_POKEMON_NAMES` Names of pokemon you want the bot to throw away regardless of IV/CP
+     * Note: `MIN_SIMILAR_POKEMON` will still be kept for all pokemon types
+   * `RELEASE_METHOD` = "CLASSIC"
+     * `KEEP_CP_OVER` Never transfer any pokemon above this CP, Setting this to 0 will never transfer anything
+     * `KEEP_IV_OVER` is the minimum pokemon IV that you want to keep (high-iv pokemons)
+     * `KEEP_IV_ONLY_WITH_PERCENT_CP` Do not keep high-iv pokemons if their CP is lower than X% of the best pokemon
+     * `MAX_POKEMON_HIGH_IV` Do not keep more than X amount of high-iv pokemons (those with highest CP are preferred by default)
+   * `RELEASE_METHOD` = "DUPLICATES"
+     * The bot seems to have a bad habit of hoarding pokemon. Enabling this feature (disabled by default) will have the bot automatically transfer pokemon that are duplicates. To determine which pokemon to transfer when duplicates exist, the pokemons are compared according to the `SCORE_METHOD` setting.  The bot will transfer the lowest scoring pokemon, maintaining `MIN_SIMILAR_POKEMON` of each type. To be completely confident that the bot will not transfer your high lvl pokemon, when this feature is enabled only pokemon with a score below `RELEASE_DUPLICATES_MAX_SCORE` are released. If you have multiple pokemon that are close to the same lvl the bot can be configured to not transfer them by using `RELEASE_DUPLICATES_SCALAR`. The value of this config is multiplied by the highest scoring pokemon of a type and only those pokemon that are less than the scaled score are transfered.
```

Comment:
can we change this "The bot will collect all pokemon it encounters , thus collecting a ton of bad ones and filling up space" since the bot does what its configured to now after your changes goes through